pkg/perf/publicip/publicip_task.go

```diff
@@ -62,6 +63,7 @@ func NewTask() perf.Task {
 		taskID:      taskID,
 		schedule:    taskSchedule,
 		description: taskDescription,
```

Comment:
i don't get what is the point of defining constants, then basically __copy__ them to the struct. The functions (say Description()) can instead return the constant directly without the need to store (copy) this to the actual task object.